tsdb/engine/tsm1/engine.go

```diff
@@ -252,6 +252,7 @@ func NewEngine(id uint64, idx tsdb.Index, path string, walPath string, sfile *ts
 
 	var planner CompactionPlanner = NewDefaultPlanner(fs, time.Duration(opt.Config.CompactFullWriteColdDuration))
 	planner.SetAggressiveCompactionPointsPerBlock(int(opt.Config.AggressivePointsPerBlock))
+	planner.SetNestedCompactor(opt.Config.NestedCompactorEnabled)
```

Comment:
perhaps log here if the config is set to true. I think that will be helpful confirmation. Sometimes we get logs but not a config file (although with Cloud1 we have direct access to both).

Reply:
Absolutely log.  Info level.  Great suggestion.